xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/BitfinexExchange.java

```diff
@@ -90,10 +87,13 @@ public void remoteInit() throws IOException, ExchangeException {
 
       if (exchangeSpecification.getApiKey() != null
           && exchangeSpecification.getSecretKey() != null) {
+        // Bitfinex does not provide any specific wallet health info
+        // So instead of wallet status, fetch platform status to get wallet health
+        Integer bitfinexPlatformStatus = dataService.getBitfinexPlatformStatus()[0];
```

Comment:
you should check if dataService.getBitfinexPlatformStatus()[0] == null, then set bitfinexPlatform status to like -1 etc

Reply:
I know to do that but i think It is not meaningful as i described in prev. comment. Using -1 value as a specific meaning(such as null) is a bad practice and it is way of doing things in 80s development environment.

Just to say, this type of meaningless comments possibly distract devs from the project.

Just doing it to make to the dev repository.

Reply:
I agree with your comment that -1 may not be the right solution but the current state is not something I will approve, we explicitly need to handle an unknown code instead of throwing an NPE.

Reply:
Yeah I fixed it with a better solution than -1 and handling NPE by using "primitive type int". It is still very bad and shitty code. I hope I don't need to deal with such issues again( We are lucky that the value is "int", I don't know what to do if it was an "Object" type). 

Anyway, you got your comment done, I hope you satisfied/persuade yourself with this meaningless comment.

Reply:
Thank you for making the changes, it is appreciated. I do also like how you solved this even more so then my suggestion.

It would be greatly appreciated if you could keep comments relevant to the discussion at hand and not bring unnecessary hostility to a PR.

Reply:
Okay, I try to do that... But as a contributor, I need to get beneficial comments given through the review process...